_im-plugin/ism/policies.md

```diff
@@ -789,6 +789,244 @@ If you want to skip rollovers for an index, set `index.plugins.index_state_manag
+                }
+              ]
+            }
+          }
+        ]
+      }
+    ],
+    "ism_template": {
+      "index_patterns": ["log*"],
+      "priority": 100
+    }
+  }
+}
+```
+
+Next, create an index template to enable the policy on:
+
+```bash
+PUT /_index_template/ism_rollover?
+```
+{% include copy-curl.html %}
+
+```json
+{
+  "index_patterns": ["log*"],
+  "template": {
+   "settings": {
+    "plugins.index_state_management.rollover_alias": "log"
+   }
+ }
+}
+```
+
+Then, change the cluster settings to trigger jobs every minute:
+
+```bash
+PUT /_cluster/settings?pretty=true
+```
+{% include copy-curl.html %}
+
+```json
+{
+  "persistent" : {
+    "plugins.index_state_management.job_interval" : 1
+  }
+}
+```
+
+Next, create a new index:
+
+```bash
+PUT /log-000001
+```
+{% include copy-curl.html %}
+
+```json
+{
+  "aliases": {
+    "log": {
+      "is_write_index": true
+    }
+  }
+}
+```
+
+Finally, add a document to the index to trigger the job:
+
+```bash
+POST /log-000001/_doc
+```
+{% include copy-curl.html %}
+
+```json
+{
+  "message": "dummy"
+}
+```
+
+You can verify these steps using the alias and index API:
+
+```bash
+GET /_cat/indices?pretty
+```
+{% include copy-curl.html %}
+
+```
+GET /_cat/aliases?pretty
+```
+{% include copy-curl.html %}
+
+#### Fail if a user tries to apply an alias action on a specific index or indexes
+
+The following policy creates an illegal argument exception with the error `"Alias action can only work on its applied index so don't accept index/indices parameter."`:
+
+```bash
+PUT /_plugins/_ism/policies/remove_action_policy?pretty
+```
+{% include copy-curl.html %}
+
+```json
+{
+  "policy": {
+    "description": "Example remove action policy.",
+    "default_state": "alias",
+    "states": [
+      {
+        "name": "alias",
+        "actions": [
+          {
+            "alias": {
+              "actions": [
+                {
+                  "add": {
+                      "alias": "log"
+                  }
+                },
+                {
+                  "remove": {
+                      "index": "log-00001",
+                      "alias": "log"
+                  }
+                }
+              ]
+            }
+          }
+        ]
+      }
+    ],
+    "ism_template": {
+      "index_patterns": ["log*"],
+      "priority": 100
+    }
+  }
+}
+```
+
+#### Fail if a user tries to remove index using alias action
+
+The following policy will create an illegal argument exception with the error `"Only ADD and REMOVE actions are allowed."`:
+
+```bash
+PUT /_plugins/_ism/policies/remove_index_policy?pretty
+```
+{% include copy-curl.html %}
+
+```json
+{
+  "policy": {
+    "description": "Example remove index policy.",
+    "default_state": "alias",
+    "states": [
+      {
+        "name": "alias",
+        "actions": [
+          {
+            "alias": {
+              "actions": [
+                {
+                  "remove_index": {
```

Comment:
We only allow add, remove, but not remove_index. Same as above, just mention in alias action section, no need to provide example.